ocaml-lsp-server/src/env_vars.ml

```diff
@@ -17,3 +17,9 @@ let _IS_HOVER_EXTENDED () : bool option =
   match v with
   | "true" | "1" -> Some true
   | _ -> Some false
+
+let _ENABLE_SYNTAX_DOCUMENTATION () : bool option =
```

Comment:
Is this config option experimental? If it's meant for end users, it shouldn't be an environment variable but a proper config option that can be passed when initializing the server.

Reply:
So, currently it works via the environment variable OR via a config option, whichever way the user chooses to activate it. 
I'd environment variables are not very convenient, i could easily refactor to just use only config options.

Reply:
Yeah, I'd drop the env var. I mostly use them to ease debugging or for experimental stuff. Users already have the LSP config system to work with. That should be enough.

Reply:
Great. The latest commit fixes this.